PULL_REQUEST_TEMPLATE

```diff
@@ -32,6 +32,16 @@ https://github.com/orgs/codice/teams
 - [ ] Documentation Updated
 - [ ] Update / Add Unit Tests
 - [ ] Update / Add Integration Tests
+
+#### Note on backporting
+Immediately following branching and prior to branch divergence, the review process for ports
```

Comment:
❓ Should we make the review rule a little bit more formal between partial and full review and approval process? I'm not sure people would know just by reading this, and communicating when to switch to full approval might be difficult.

A few suggestions:
* Partial review and approval is only allowed right after a release and up until the first patch version is released
* Partial review and approval is only allowed when there have been less than _N_ commits or PRs against the release branch
* We update the template when a release branch is created to indicate that only partial approval is required, and change it back to the full approval process when we feel that the release branch and master have diverged

Reply:
I don't know that we can make it more defined and not actually be wrong. I could easily foresee a time when we cut a branch for release and work for master that had intentionally been held back is immediately merged causing the branches to diverge immediately. Number of commits or PRs against the release branch isn't the gating factor, it's the commits that only went to its parent branch. And even then, I believe there should be a small degree of judgment applied. Examples:

1. Branch X is created. A brand new feature is immediately committed to master that does not interact with any existing features. I think this case might safely allow for forward-ports with abbreviated review.
2. Branch X is created. A large number of documentation-formatting/layout updates are immediately committed to master. I believe that any bug fixes made to branch X (that involve no updates to documentation) could safely be ported to master with abbreviated review.
3. Branch X is created. No new features are added to master through three subsequent patch releases of X. There is definitely no risk of abbreviated review in this case.
3. Branch X is created. A change is immediately made to an existing feature in master, causing the branches to diverge. I think full review ports from X are now clearly required.

I do like your final suggestion, that we make a template update when we have determined that the branches have diverged and require the abbreviated process to be set aside. The only problem I see with that is that any open port PRs will still have the old template. But this can be managed by committers who'll be aware of the status change and know they're looking at a PR with outdated metadata. 

I'll take a swipe at indicating this on the template and will also add something about metadata-type PRs that can always undergo abbreviated review. This PR, for example, does NOT need static analysis or CI as it only impacts the metadata of the repository.